scipy/special/Faddeeva.cc

```diff
@@ -118,34 +118,15 @@
 */
 
 #include <Python.h>
-extern "C" {
-#include <numpy/npy_math.h>
-}
-
 #include <cfloat>
 #include <cmath>
 
 #define complex std::complex
 
 /////////////////////////////////////////////////////////////////////////
 
-// use numpy's versions, since std:: versions only available in C++11
-#define my_copysign npy_copysign
-#define Inf NPY_INFINITY
-#define NaN NPY_NAN
-
-/*
- * portable isnan/isinf; in cmath only available in C++11 and npy_math
- * versions don't work well (they get undef'd by cmath, see gh-5689)
- * Implementation based on npy_math.h
- */
-#define my_isnan(x) ((x) != (x))
-#ifdef _MSC_VER
-    #define my_isfinite(x) _finite((x))
-#else
-    #define my_isfinite(x) !my_isnan((x) + (-x))
-#endif
-#define my_isinf(x) (!my_isfinite(x) && !my_isnan(x))
+#define Inf INFINITY
+#define NaN NAN
```

Comment:
These defines seem pretty pointless now. I mean, this could be a mechanical cleanup also later, but I think it'd be a good time in this PR too.

Reply:
This is used quite extensively and there's upstream code, so I decided to leave it alone.